les/fetcher.go

```diff
@@ -116,6 +117,7 @@ func newLightFetcher(pm *ProtocolManager) *lightFetcher {
 		timeoutChn:     make(chan uint64),
 		requestChn:     make(chan bool, 100),
 		syncDone:       make(chan *peer),
+		SyncedCh:       make(chan bool, 1),
```

Comment:
You define a new channel -- but I don't see where you use it?

Reply:
Sorry that was deprecated when i switched to useing the downloader.done event, it has been removed.